### ISSUE TITLE
wrong location of itos.pkl

### ISSUE THREAD

**Issue by dalupus** [tpietruszka/ulmfit_attention#1]
The actual location appears to now be in either the tmp_lm or tmp_clas directories.  There is no itos.pkl in the imdb directory
![fastai-dev - VMware Workstation 2020-04-22 11 43 50](https://user-images.githubusercontent.com/806458/80003213-b11e5900-848e-11ea-979d-7cde52f06b52.png)

https://github.com/tpietruszka/ulmfit_attention/blob/12cc30656c35eeb2fc7361b9052b48d00c9014ea/ulmfit_attention/datasets.py#L27

**Comment by dalupus** [comment]
I'm an idiot and didn't read the instructions :)

**Comment by tpietruszka** [comment]
Hey @dalupus , no worries, thank you for testing the code! 
Does it work after all?
If so, I think I should close #2 ?

**Comment by dalupus** [comment]
sorry thought I had closed that.

Yes, everything worked as expected.  Although I am curious exactly how the encoder was trained.

**Comment by tpietruszka** [comment]
I simply used the [ULMFiT example notebook](https://github.com/fastai/fastai/blob/master/examples/ULMFit.ipynb), likely with a lower batch size to avoid running out of memory.

I cannot find the exact notebook now - my bad -  but there was nothing more to it.

**Comment by dalupus** [comment]
The reason I am curious, is I've not had luck with ULMFiT with lower batch sizes.  It seems the lower the batch size the worse the results (no matter how long I train).  Which sucks since I actually have some 1070's sitting around doing nothing.  Instead, I am having to train on a p3.2xlarge up in amazon.

Basically I've seen a linear relationship between batch size and performance.  Maybe is just because some other hyperparameter needs to be tuned to deal with lower batch sizes.

**Comment by tpietruszka** [comment]
Interesting. I think my results did not change too much, except if I had to lower BPTT. I did adjust the learning rate AFAIR.

Seems like I dropped the ball here, regarding reproducibility. Although I still hope I will be able to dig up the notebook I used.

Just to make sure -  was the relationship between bs and performance you saw:
1. during language model pre-training/fine-tuning, in terms of perplexity
2.  during language model pre-training/fine-tuning, in terms of performance of the downstream classifier
3. during classifier training

or perhaps all of the above?

Did you adjust the learning rate in any way?
